content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Response
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and whites are superior to all other ethnicities (racism).  
+>   
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them special.  
+>   
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’. 
+For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice.  
+>  
+> Even if *it is the case* we are uniquely superior, morality requires we care for those less capable than ourselves; not that we treat them as a resource.
+
+## Detail - Analytic
+
+> A Definition of Superior (they are many and varied) : of higher rank, quality, or importance - [Merriam-Webster](http://web.archive.org/web/20190711165845/https://www.merriam-webster.com/dictionary/superior)
+
+
+What most people forget (*or choose to overlook*) in using the term superior is that:  
+
+
+1. we determine the ranking systems, 
```

Comment:
Shall we make these independent sentences (i.e. each with a capital/full stop) rather than a single sentence punctuated by numbered list items? As a website it looks a little quirky as is.